index.html

```diff
@@ -64,5 +65,192 @@ <h2><a href="https://www.linkedin.com/in/ivan-martinez-von-halle/" class="connec
+          <img src="./images/first-page-img/medium.svg" alt="medium logo">
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/ivanmvh" aria-label="GitHub">
+          <img src="./images/first-page-img/github.svg" alt="github logo">
+        </a>
+      </li>
+      <li>
+        <a href="" aria-label="Angel list">
+          <img src="./images/first-page-img/peace.svg" alt="Angel list logo">
+        </a>
+      </li>
+    </ul>
+    <button class="resume-btn">Get my resume</button>
+  </div>
+
+  <div class="about-languages">
+    <div class="languages-section">
+      <h3 class="languages">Languages</h3>
+      <img class="arrow" src="./images/languages-img/down-arrow.svg" alt="down-arrow" height="15" width="15">
+    </div>
+    <div class="languages-boxes">
+      <div class="language-box">
+        <img src="./images/languages-img/JS.svg" alt="javascript image" height="50" width="50">
+        <p>JavaScript</p>
+      </div>
+      <div class="language-box">
+        <img src="./images/languages-img/html.svg" alt="html image" height="50" width="50">
+        <p>HTML</p>
+      </div>
+      <div class="language-box">
+        <img src="./images/languages-img/css.svg" alt="css image" height="50" width="50">
+        <p>CSS</p>
+      </div>
+    </div>
+    <div class="languages-section frameworks">
+      <h3 class="languages">Frameworks</h3>
+      <img class="arrow" src="./images/languages-img/right-arrow.svg" alt="right-arrow" height="15" width="15">
+    </div>
+    <div class="languages-section skills">
+      <h3 class="languages">Skills</h3>
+      <img class="arrow" src="./images/languages-img/right-arrow.svg" alt="right-arrow" height="15" width="15">
+    </div>
+  </div>
+ 
+ </section>
+ <!-- about section (end) -->
+ 
+ <!-- contact section (init) -->
+<div class="contact-background" loading="lazy">
+  <section id="contact" loading="lazy">
+   <h3 class="contact-header">Contact me</h3>
+   <p class="contact-desc">If you have an application you are interested in developing, a feature that you need built
+     or a project that needs coding. I'd love to help with it </p>
+  
+   <form action="https://formspree.io/f/mzbwzdgd" class="form-container" method="post" onsubmit="return formSubmit();">
+     <input type="text" name="name" placeholder="Your name" maxlength="30" required class="text-input" id="name">
+     <input type="email" name="email" placeholder="Your email" required class="email-input" id="email">
+     <textarea placeholder="Write your message here" maxlength="500" name="message" id="message" cols="30" rows="10"
+       required></textarea>
+       <p class="error">The email should be lower case</p>
+     <button type="submit" class="contact-btn">Get It Touch</button>
+   </form>
+   <div>
+    <img src="./images/contact-img/Blue_line_bottom.svg" alt="blue line" height="15" width="100">  
+  </div>
+  </section>
+  </div>
+  <!-- contact section (end) -->
```

Comment:
Great job building this section. It has just made your next week  easier. Kuddos. 👏🏾